### PR TITLE
Doc: added "Arguments" removing implicit arguments

### LINKED ISSUE #13283: Improve error message when "Arguments ... : clear implicit" is used with implicit annotations.
Sometimes, one wants to change which arguments of a constant are implicit which are not.

I just found out about the "rename" command because I wanted `clear implicit` to behave like `rename`. Before I always called Arguments twice: once to clear implicits, and once to set new.

Maybe one could improve the error message here:

````
Require Import Fin.

Fail Arguments Fin.of_nat_lt _ {_} _: clear implicits.
Arguments Fin.of_nat_lt _ {_} _: rename.
````

The Error is: `The "clear implicits" flag is incompatible with implicit annotations`
It could be: `The "clear implicits" flag is incompatible with implicit annotations. Use "rename" flag instead`

Alternatively, one could allow the failing command from above, so that `clear implicit` behaves like `rename`.

### PR DESCRIPTION
**Pull request by fakusb**
**Kind:** documentation

closes #13283

### CODE REVIEW

File: doc/sphinx/language/extensions/arguments-command.rst [hunk 1]
```diff
@@ -86,6 +86,7 @@ Setting properties of a function's arguments
          the parameter name used in the function definition).  Unless `rename` is specified,
          the list of :n:`@name`\s must be a prefix of the formal parameters, including all implicit
          arguments.  `_` can be used to skip over a formal parameter.
+         This declares the name as non-implicit if either `clear implicits` is declared or at least one other name is declared implicit in the same list of :n:`@name`\s.
```
Comment (jfehrle) [inline doc/sphinx/language/extensions/arguments-command.rst:89]:
```suggestion
         This declares :n:`@name` as non-implicit if `clear implicits` is specified or at least one other name is declared implicit in the same list of :n:`@name`\s.
```

A little unclear what "This" refers to.  If it's the previous sentence, maybe splice the sentences:  `... to skip over a formal parameter, which declares ...`  If it's the construct, maybe say `This construct declares the name ...`

**Review by jfehrle — COMMENTED** [review]
Thanks for the PR.  Some wording suggestions.  I'll let someone else review it for technical accuracy.

File: doc/sphinx/language/extensions/arguments-command.rst (hunk 1 above)
Comment (fakusb) [reply doc/sphinx/language/extensions/arguments-command.rst]: I made it more clear: https://github.com/coq/coq/pull/13293/commits/87be983b7623126320dc0b43303a48ea41b86f5c#

**Review by Zimmi48 — APPROVED** [review]
Thanks!

**Comment by Zimmi48** [comment]
@jfehrle I'll let you merge if you're happy with the wording improvement.

**Review by jfehrle — APPROVED** [review]
LGTM

**Comment by jfehrle** [comment]
Doc change only, CI failures are unrelated.

@coqbot: merge now

**Pull request merged**